src/pybind/mgr/nfs/export.py

```diff
@@ -803,14 +825,24 @@ def _apply_export(
                 log.debug("export %s pseudo %s -> %s",
                           old_export.export_id, old_export.pseudo, new_export_dict['pseudo'])
 
+        fsal_dict = new_export_dict.get('fsal')
+        if fsal_dict and fsal_dict['name'] == NFS_GANESHA_SUPPORTED_FSALS[0]:
+            # Ensure cmount_path is present in CephFS FSAL block
+            if not fsal_dict.get('cmount_path'):
+                if old_export:
+                    new_export_dict['fsal']['cmount_path'] = old_export.fsal.cmount_path
```

Comment:
So, it is always a surety to have `old_export.fsal.cmount_path` set to something?

Reply:
Yes! That's what I explained before as well that even if nothing specified from code we'll still set it as `/` atleast

Reply:
if that is the case (code defaulting to `/`) then do we really need the else block to assign `/` (again)?

Reply:
yes, that's why I added a line to assig it `/` value